units/systemd-tmpfiles-setup-var-log-journal.service.in

```diff
@@ -0,0 +1,23 @@
+#  SPDX-License-Identifier: LGPL-2.1+
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=Set ACLs and DACs for /var/log/journal
+Documentation=man:tmpfiles.d(5) man:systemd-tmpfiles(8)
+DefaultDependencies=no
+Conflicts=shutdown.target
+After=local-fs.target systemd-sysusers.service systemd-journald.service
+Before=sysinit.target shutdown.target
```

Comment:
Is Before=sysinit.target needed?  This transatively adds that dependency to systemd-journal-flush.service delaying boot.

Reply:
It's already in systemd-tmpfiles-setup - the list of dependencies was copied from there. So there's no change from the point of view of systemd-journal-flush.

https://github.com/systemd/systemd/blob/master/units/systemd-tmpfiles-setup.service#L16

Reply:
To answer your finer question - I do not know whether it is necessary or not. Simply copied the existing behaviour.